friendshipper/src-tauri/src/system/terminal.rs

```diff
@@ -0,0 +1,24 @@
+use std::process::Command;
+
+use axum::extract::State;
+use tracing::error;
+
+use crate::engine::EngineProvider;
+use crate::state::AppState;
+
+pub async fn open_terminal_to_path<T>(State(_state): State<AppState<T>>, path: String)
+where
+    T: EngineProvider,
+{
+    if let Err(e) = Command::new("cmd")
```

Comment:
i think we'll need to guard this with `#[cfg(windows)]`, that way we can add a separate path for mac/linux etc.